shopinvader_api_payment_cart/models/sale_order.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2024 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class SaleOrder(models.Model):
+
+    _inherit = "sale.order"
+
+    def action_confirm_cart_from_tx(self, tx):
```

Comment:
```suggestion
    def _action_confirm_cart_from_tx(self, tx):
```

This method does not need to be public